image_setup/04_save_release/build.bash

```diff
@@ -7,6 +7,11 @@ if [ "$DEFAULT_EXECMODE" = "devel" ]; then
     exit 1
 fi
 
+TARGETPATH="."
```

Comment:
In general definitely a good idea to allow passing the target path as an argument.

Only thing that I'd like to add to this script is a directory/file existence check to avoid accidental overwriting of previous exports. Either with error and exit straight away or with a user prompt [y/n].

Reply:
The export name is time-tagged, you'd have to start the export the same minute, in that case you most probably want to overwrite.

But there should be a folder check...

Reply:
@planthaber 

Okay, I see.

For the folder check I would mkdir the directory in case it doesnt exist and warn/exit if it already exists.

Reply:
Thought about that but decided against the auto-creation because it could by copy/paste of the wrong text